src/main/java/app/linguistai/bmvp/model/gamification/quest/types/CreateWordListCriteria.java

```diff
@@ -1,8 +1,7 @@
 package app.linguistai.bmvp.model.gamification.quest.types;
 
-import lombok.Data;
-
-@Data
 public class CreateWordListCriteria extends QuestCompletionCriteria {
-    private Integer times;
+    public CreateWordListCriteria(Integer times) {
```

Comment:
This quest is never added. 
A quest for adding x amount of unknown words might be better - it is much more repeatable than adding word lists.

Reply:
Yes, this is just a placeholder for the add x amount of unknown words quest. It will not be developed for this sprint, however, it would be better to change the name to reflect this (name remained from previous iteration). I will rename this file